pkgs/by-name/pr/prometheus-restic-exporter/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, python3
+, restic
+, nixosTests
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "prometheus-restic-exporter";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "ngosang";
+    repo = "restic-exporter";
+    rev = version;
+    hash = "sha256-Qwhlecginl5+V+iddN/vIHfJA1kQOZtscECsoD4LJPE=";
+  };
+
+  nativeBuildInputs = [
+    (python3.withPackages (ps: [ ps.prometheus-client ]))
+  ];
```

Comment:
Does it not need python + prometheus-client during runtime? This seems like it should be `buildInputs`.